src/assets/CollectiblesController.ts

```diff
@@ -236,14 +237,27 @@ export class CollectiblesController extends BaseController<
   ): Promise<CollectibleMetadata> {
     const tokenURI = this.getCollectibleApi(contractAddress, tokenId);
     let collectibleInformation: ApiCollectible;
-
     /* istanbul ignore if */
-    if (this.openSeaApiKey) {
-      collectibleInformation = await handleFetch(tokenURI, {
-        headers: { 'X-API-KEY': this.openSeaApiKey },
-      });
-    } else {
+    try {
       collectibleInformation = await handleFetch(tokenURI);
+    } catch (e) {
+      logOrRethrowError(e);
+
+      if (this.openSeaApiKey) {
+        collectibleInformation = await handleFetch(
+          `https://api.opensea.io/api/v1/asset/${contractAddress}/${tokenId}`,
+          {
+            headers: { 'X-API-KEY': this.openSeaApiKey },
+          },
+        );
+      } else {
+        return {
+          name: null,
+          description: null,
+          image: null,
+          standard: null,
```

Comment:
What is this property for? It's not returned on the other paths

Reply:
returning out here allows us to skip that and the re-assigning step and just use the more easily encoded default/null version of the `CollectibleMetadata` which is ultimately return type of this function...

Reply:
Yep! I must have missed it